index.html

```diff
@@ -330,7 +440,7 @@ <h3>
             section "Generating entropic identifiers for DID documents" for more
```

Comment:
signing first and forcing JWS signatures to be expressed as proof objects LD-style will be unpopular and confusing to the JSON-only crowd but seems a reasonable compromise, since it allows multiple proofing formats insofar as the wierd "deconstructed" JWT is at least convertable into a real JWT